tensornetwork/backends/jax/jitted_functions.py

```diff
@@ -59,9 +59,8 @@ def jax_lanczos(matvec, arguments, init, ncv, neig, landelta, reortho):
     def body_modified_gram_schmidt(i, vals):
       vector, krylov_vectors = vals
       v = krylov_vectors[i, :]
-      vector -= jax.numpy.dot(jax.numpy.conj(v),
-                              jax.numpy.ravel(vector)) * jax.numpy.reshape(
-                                  v, vector.shape)
+      vector -= jax.numpy.vdot(v, jax.numpy.ravel(vector)) * jax.numpy.reshape(
+          v, vector.shape)
```

Comment:
Weren't we using jnp? `ravel` is also redundant inside `vdot`. What about

`vector -= jnp.vdot(v, vector).reshape(vector.shape)`

Reply:
jnp is only used within the JaxBackend

Reply:
Fair enough. `vdot` still flattens by definition, though.

Reply:
removed `ravel`